### DISCUSSION TITLE
how to share connection of tun2socks in windows

### DISCUSSION THREAD

**Discussion by marthaboursaw72** [xjasonlyu/tun2socks#239] [General]
i am using windows 10 and basic idea behind is to make my windows 10 hotspot and then share tun2socks internet to hotspot so i can use socks 5 on my android phone... but my android phone is showing no internet available while if i am trying to share openvpn vpn service to same hotspot its working on my android phone so how do i share socks 5 over hotspot created by tun2socks?

i didnt made any changes in route as i dont want to use socks in my windows system i am jst trying to use it on android.. and devices connected on hotspot

**Comment by xjasonlyu** [comment]
Have you enabled ip forwarding on your windows server?

**Reply by marthaboursaw72** [reply]
its not windows server.. its normal windows 10

**Reply by xjasonlyu** [reply]
yeah, the same.

**Reply by marthaboursaw72** [reply]
> yeah, the same.

just enabled it still not working.. (ENABLED FROM REGEDIT I AM NOT SURE IF I HAVE TO DO SOMETHING ELSE IF YES PLEASE EXPLAIN)
